addons/l10n_ua/data/account_account_tag_data.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="acc_tag_vat" model="account.account.tag">
```

Comment:
It does not seem used

Reply:
Right, it is used in vat addon. I will remove it.

Reply:
Are the account tags of type taxes also used in the vat addon?  Because then maybe they could stay here as they make sense.  You never know we could agree on putting it in standard too.

Reply:
Yes, tags on taxed are also used to know which taxes are vat.